### PR TITLE
Added ansible fix for rsyslog_remote_loghost

### PR DESCRIPTION
**Pull request by mpreisler**

### CODE REVIEW

**Comment by mpreisler** [comment]
Give me a sec, this needs `create=yes` I think. Will check.

File: shared/templates/static/ansible/rsyslog_remote_loghost.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+
+- name: "Set rsyslog remote loghost to (ansible-populate rsyslog_remote_loghost_address)"
+  lineinfile:
+    dest: /etc/rsyslog.conf
+    regexp: "^\\*\\.\\*"
+    line: "^\\*\\.\\* @@(ansible-populate rsyslog_remote_loghost_address)"
```
Comment (yuumasato) [inline shared/templates/static/ansible/rsyslog_remote_loghost.yml]: There is no need for `^` and `\\`, the line is added as is.

Comment (mpreisler) [inline shared/templates/static/ansible/rsyslog_remote_loghost.yml]: Good catch, fixed.

**Comment by yuumasato** [comment]
Ack.

**Pull request merged**